build/common.props

```diff
@@ -4,7 +4,7 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>28</MinorVersion>
+    <MinorVersion>99</MinorVersion>
```

Comment:
Bumped version so that we can can be certain we are using the right build when checking the logs.